__tests__/fixtures/react-native/fabricRefRewrites/reactDevelopment/output.js

```diff
@@ -712,10 +723,11 @@ if (process.env.NODE_ENV !== 'production') {
       const isTurboModuleEnabled = global.RN$Bridgeless || global.__turboModuleProxy != null;
       if (isTurboModuleEnabled && Platform.OS === 'ios') {
         if (
-          type.$$typeof &&
-          (type.$$typeof.toString() === 'Symbol(react.forward_ref)' ||
-            type.$$typeof.toString() === 'Symbol(react.element)' ||
-            type.$$typeof.toString() === 'Symbol(react.transitional.element)')
+          isReact19Plus() ||
```

Comment:
In React 19 with React Native 81, `type.$$typeof` no longer exists. Do not check for `$$typeof` in React 19.